src/StellarApi/StellarApi.EntityToModel/MapExtensions.cs

```diff
@@ -0,0 +1,51 @@
+using StellarApi.Entities;
+using StellarApi.Model.Space;
+
+namespace StellarApi.EntityToModel;
+
+/// <summary>
+/// Extension methods for converting between <see cref="Map"/> and <see cref="MapEntity"/>.
+/// </summary>
+public static class MapExtensions
+{
+    /// <summary>
+    /// Converts a <see cref="MapEntity"/> to a <see cref="Map"/>.
+    /// </summary>
+    /// <param name="entity">The entity to transform.</param>
+    /// <returns>The new Map object.</returns>
+    public static Map ToModel(this MapEntity? entity)
+        => entity is null ? null : new Map(entity.Id, entity.Name, entity.CreationDate, entity.ModificationDate);
+
+    /// <summary>
+    /// Converts a <see cref="Map"/> to a <see cref="MapEntity"/>.
+    /// </summary>
+    /// <param name="model">The model to transform.</param>
+    /// <returns>The new entity Map object.</returns>
+    public static MapEntity ToEntity(this Map? model)
+        => model is null
+            ? null
+            : new MapEntity
```

Comment:
Provide a way to also add celestial objects in the model using the existing extension methods for this type.